python/doc/pyplots/GaussianProcessFitter.py

```diff
@@ -0,0 +1,24 @@
+import openturns as ot
+from openturns.experimental import GaussianProcessFitter
+from openturns.viewer import View
+
+f = ot.SymbolicFunction(["x"], ["x * sin(x)"])
+sampleX = [[1.0], [2.0], [3.0], [4.0], [5.0], [6.0], [7.0], [8.0]]
+sampleY = f(sampleX)
+basis = ot.Basis(
+    [ot.SymbolicFunction(["x"], ["x"]), ot.SymbolicFunction(["x"], ["x^2"])]
+)
+covarianceModel = ot.SquaredExponential([1.0])
+covarianceModel.setActiveParameter([])
+algo = GaussianProcessFitter(sampleX, sampleY, covarianceModel, basis)
+algo.run()
+result = algo.getResult()
+
+graph = f.draw(0.0, 10.0)
+graph.add(result.getMetaModel().draw(0.0, 10.0))
```

Comment:
If possible, it would be more interesting to represent the confidence intervals of the GP, or else a bunch of realizations. The GP unconditioned mean function is by itself of little interest.

Reply:
This is the fitter algo (thus a general linear model only)

Reply:
OK, but I still think this feature on the Fitter would be interesting. It is not mandatory for this PR however.